styleguide/source/assets/scss/02-molecules/_article-metadata.scss

```diff
@@ -1,12 +1,6 @@
-.ama__topic_article-preview_metadata {
+.ama__article-stub__metadata {
```

Comment:
Seems like this should probably live in `_article-stub.scss` instead of having another partial for the metadata

Reply:
the article stub metadata has it's own css so it has parity with the twig file called article-metadata.twig

Reply:
Because our css gets balled into one. I don't think having the meta css is going to be an issue as files sizes go. I rather have separate files per twig file if that makes sense.

Reply:
@designerneil I think the issue here is the classname. If the class name is `ama__article-stub__metadata` therefore saying "this thing belongs to `article-stub`" then it should be included within the `_article-stub.scss`. If you want to keep the `scss` files to align with the `twig` files then that's fine, but the class name should also be updated to relfect that. Therefore the class should change to `ama__metadata` in that case. This would no longer make it a child element of the `article-stub`.

Reply:
popping in to say that in addition to the above a class of `ama__article-metadata` should be added as a top-level wrapper to `article-metadata.twig`, and any includes of that pattern should adjusted accordingly. Then, styling for the article metadata pattern can go in `_article-metadata.scss`.

Reply:
The reason why the metadata doesn't have a wrapper is because D8 is consuming as two separate entities: 
field--node--field-publish-date--news-article.html.twig 
field--node--field-read-time--news-article.html.twig

without a parent container that contains both the metadata entities